trade-finance-manager-api/src/v1/rest-mappings/mappings/gef-facilities/mapGefFacilitySnapshot.js

```diff
@@ -0,0 +1,64 @@
+const { formattedNumber } = require('../../../../utils/number');
+const mapFacilityValue = require('../facilities/mapFacilityValue');
+const mapFacilityProduct = require('../facilities/mapFacilityProduct');
+const mapFacilityType = require('../facilities/mapFacilityType');
+const mapGuaranteeFeePayableToUkef = require('../facilities/mapGuaranteeFeePayableToUkef');
+const mapGefUkefFacilityType = require('./mapGefUkefFacilityType');
+const mapGefFacilityDates = require('./mapGefFacilityDates');
+const mapFacilityValueExportCurrency = require('../facilities/mapFacilityValueExportCurrency');
+const mapUkefExposureValue = require('../facilities/mapUkefExposureValue');
+const { mapGefFacilityStage } = require('../facilities/mapFacilityStage');
+
+/**
+ * Maps the existing GEF facility snapshot in the database to the facility snapshot used in TFM API.
+ * This function is only used on GEF facilities.
+ * Note: This implementation modifies the facility snapshot to have values not consistent with the facility snapshot in the database.
+ * In particular, this is a live object that updates e.g. when amendments are added in TFM.
```

Comment:
I wouldn't say live object, as that might infer we are mutating this object automagically on updating facilities :)

I'd probably say something as per my other comments about applying TFM updates to the GEF facility or similar

Reply:
Good idea, have changed all the comments